profiles/default/linux/make.defaults

```diff
@@ -13,9 +13,6 @@
 # Default starting set of USE flags for all default/linux profiles.
 USE="crypt ipv6 ncurses nls pam readline ssl zlib"
 
-# make sure toolchain has sane defaults <toolchain@gentoo.org>
-USE="${USE} fortran openmp"
```

Comment:
maybe also remove these flags from

- `prefix/linux/make.defaults`
- `default/linux/make.defaults`
- `arch/mips/make.defaults`

Reply:
Added a commit for prefix/linux.
default/linux was done already.
arch/mips *disables* fortran and openmp, so I left it alone.